ee/modules/650-runtime-audit-engine/openapi/config-values.yaml

```diff
@@ -1,2 +1,6 @@
 type: object
-properties: {}
+properties:
+  debugLogging:
+    type: boolean
+    description: |
+      If true all outputs will print in stdout the payload they send.
```

Comment:
i think it does not match with russian doc

Reply:
done